clusterloader2/pkg/measurement/common/probes/probes.go

```diff
@@ -173,6 +173,13 @@ func (p *probesMeasurement) start(config *measurement.Config) error {
 		return err
 	}
 	if err := p.waitForProbesReady(config); err != nil {
+		if err.Error() == wait.ErrWaitTimeout.Error() {
+			// Start the measurement nonetheless as timing out during waiting for
+			// probe pods to be ready is almost always because of a small fraction
+			// of pods not visible via Prometheus.
+			p.startTime = time.Now()
```

Comment:
Setting it in two places separately calls for inconsistencies in the future.

lets restructure:

	err := waitForProbes
	if err != nil && err is fundamental {
		return fmt.Errorf(err)
	}
	// comment that most of the probes started
	startTime = now
	if err != nil {
		return fmt.Errorf(err)
	}
	return nil